### ISSUE TITLE
TOTP on checkbox

### ISSUE THREAD

**Issue by bwbroersma** [keepassxreboot/keepassxc-browser#797]
Example from [Uptime Robot](https://uptimerobot.com/) (related to #768):
![image](https://user-images.githubusercontent.com/4252918/75884277-d09cec00-5e24-11ea-8a88-086e287f3641.png)

```html
<input id="showQR" class="checkbox" type="checkbox" value="0" name="userAuthSetting">
```
Because of the `Auth` in the name, it's detected as TOTP field, while it is a checkbox.
## Expected Behavior
No Fill in TOTP on a checkbox.

## Current Behavior
Fill in TOTP overlay over a checkbox making it undetectable (I had to disable the addon to interact with the input).

## Possible Solution
There is a size check underway which would probably fix this specific issue, but not the underlying issue that there currently is no type checking:
https://github.com/keepassxreboot/keepassxc-browser/blob/280a561bb0276fec0ff5c5af37bd61e7cce5fb13/keepassxc-browser/content/keepassxc-browser.js#L1031-L1041

only a negative check on password:
https://github.com/keepassxreboot/keepassxc-browser/blob/280a561bb0276fec0ff5c5af37bd61e7cce5fb13/keepassxc-browser/content/totp-field.js#L30-L40

Probably only text or number or empty/blank types should be whitelisted, instead of password being blacklisted.

## Debug info
KeePassXC - 2.5.3
KeePassXC-Browser - 1.5.4
Operating system: Linux x86_64
Browser: Mozilla Firefox 75.0

**Comment by droidmonkey** [comment]
This will be fixed in 1.6.0 of the browser extension, already merged into develop.

**Comment by varjolintu** [comment]
Some sites actually use `tel` type for TOTP, so that must not be ignored.
